test/Microsoft.Azure.WebJobs.Host.UnitTests/Blobs/BlobPathTests.cs

```diff
@@ -8,9 +8,24 @@
 
 namespace Microsoft.Azure.WebJobs.Host.UnitTests.Blobs
 {
-    public class BlobPathTests 
+    public class BlobPathTests
     {
         [Fact]
+        public void CreateBlobWithValidPattern()
```

Comment:
You should also add some E2E tests to the existing BlobBindingEndToEndTests suite. E.g. once SAS URLs and public URLs are supported, have an e2e test demonstrating both.